clients/drcachesim/tools/view.cpp

```diff
@@ -117,13 +117,10 @@ bool
 view_t::parallel_shard_exit(void *shard_data)
 {
     // If the framework exited (-exit_after_records, e.g.), be sure to print
-    // out a delayed timestmap.
+    // out a delayed timestamp.
     if (timestamp_ > 0) {
         memtrace_stream_t *memstream = reinterpret_cast<memtrace_stream_t *>(shard_data);
-        memref_t memref;
-        memref.marker.type = TRACE_TYPE_MARKER;
-        memref.marker.marker_type = TRACE_MARKER_TYPE_TIMESTAMP;
-        memref.marker.tid = timestamp_tid_;
+        memref_t memref = timestamp_memref_;
```

Comment:
Can avoid the copy and directly use timestamp_memref_ below?

Reply:
Done.